src/hooks/useRuntime.ts

```diff
@@ -40,7 +39,8 @@ interface CarouselTrace {
   cards: CardTrace[];
 }
 
-export interface RuntimeOptions extends SetOptional<VoiceflowRuntimeOptions<RuntimeContext>, 'url'> {
+export interface RuntimeOptions extends Omit<VoiceflowRuntimeOptions<RuntimeContext>, 'url'> {
```

Comment:
it's very strongly typed so `SetOptional` was just `url?: string` which is not equivalent to `url?: string | undefined`